todoapp/todoapp/settings.py

```
@@ -88,7 +88,7 @@
 DATABASES = {
     u'default': {
         u'ENGINE': u'django.db.backends.postgresql',
-        u'NAME': u'',                      # Add database name.
+        u'NAME': u'mydjangodb',                      # Add database name.
```

Ruff (0.12.2)

Remove unicode literals from strings

Remove unicode prefix

(UP025)